cnxeasybake/oven.py

```diff
@@ -386,7 +402,7 @@ def build_recipe(self, element, step, depth=0):
             if 'outside_deferred' in matching_rules:
                 for rule, declarations in \
                         matching_rules.get('outside_deferred'):
-                    logger.debug(u'Rule ({}): {}'.format(*rule))
+                    self.report_coverage(rule)
```

Comment:
Actually, the matching rules are returned and iterated over at L271, when they're put into the pseudo-element dictionary. hould be able to do w/ one call to report_coverage (perhaps renamed `record_coverage()` ?)